app/presenters/arclight/show_presenter.rb

```diff
@@ -9,5 +9,22 @@ def heading
       title += ", #{date.first}" if date.present?
       title
     end
+
+    def with_field_group(group)
+      self.field_group = group
+      self
+    end
+
+    private
+
+    def field_group
+      @field_group || 'show_field'
+    end
+
+    attr_writer :field_group
+
+    def field_config(field)
+      BlacklightFieldConfigurationFactory.for(config: configuration, field: field, field_group: field_group)
```

Comment:
Interested in folks' take on this.  This is in response to https://github.com/projectblacklight/blacklight/blob/master/app/presenters/blacklight/show_presenter.rb#L89-L91, which explictly uses `config.show_fields`.  This allows us to pass in a `field_group` that can be used to determine which configuration to use.

Without this, we can't access any configuration options that we would want to use in Blacklight's rendering pipeline such as `separator_options` (used in this PR), `helper_method`, `link_to_facet` etc.

Another option would be to write presenters for each field group, but that felt, eh.